components/tools/OmeroWeb/omeroweb/webgateway/static/webgateway/js/ome.plateview.js

```diff
@@ -132,13 +134,15 @@ jQuery._WeblitzPlateview = function (container, options) {
     // NB: don't add other classes here - will get removed on slider change.
     table.addClass('showWellLabel wellSize' + opts.width);
 
+    var imgIds = new Array();
```

Comment:
Much prefer the array literal ``` var imgIds = [];```
E.g. https://www.w3schools.com/js/js_arrays.asp
"There is no need to use new Array(). For simplicity, readability and execution speed, use the array literal method."
Or the airbnb js style guide: https://github.com/airbnb/javascript
```
// bad
const items = new Array();
// good
const items = [];
```